modules/services/repositories/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
+    <uses-permission android:name="android.permission.SCHEDULE_EXACT_ALARM" />
```

Comment:
I don't think we really have to have this since this is taken care of by the app module, but it is probably a good idea to include it here as well and removes a lint warning from Android Studio.